releasenotes/notes/feature-sci-git-metadata-0b418d0a3529d11f.yaml

```
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Extract git metadata from environment variables (DD_GIT_REPOSITORY_URL, DD_GIT_COMMIT_SHA),
+    or from the python package specified by the DD_MAIN_PACKAGE.
+    Send git metadata via _dd.git.repository_url and _dd.git.commit.sha root span tags for tracer,
+    and via git.repository_url and git.commit.sha tags for profiler
+    It can be disabled by the DD_TRACE_GIT_METADATA_ENABLED
```

I don't think the `_dd.git....` tags are user facing. Correct me if I am wrong but I believe they will be only used internally. Feel free to ignore or edit the suggestion below

```suggestion
    sci: Extracts and sends git metadata from environment variables ``DD_GIT_REPOSITORY_URL``, ``DD_GIT_COMMIT_SHA``,
    or ``DD_MAIN_PACKAGE``.  This feature can be disabled by setting ``DD_TRACE_GIT_METADATA_ENABLED=False``.
```

---

Thanks! Done.
(Just little bit updated, because we extract values not from environment variable DD_MAIN_PACKAGE, but form the package with this name)